keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -336,22 +345,23 @@ type IngressBackendHTTPHealthcheck struct {
 
 If `Healthcheck` is nil, then the implementation default healthcheck will be
 configured, healthchecking the root `/` path. If `Healthcheck` is specfied,
-then the backend health will be checked using the parameters listed above. 
+then the backend health will be checked using the parameters listed above.
 
 ### Status
 
-!!! TODO
+As this is strictly additive, this can be punted to post-GA to reduce
+the size of the change.
 
 ### Ingress class
 
 The `kubernetes.io/ingress.class` annotation is required for interoperation
 between various Ingress providers. As support for this annotation is universal,
 this concept should be promoted to an actual field.
 
-#### Proposal
+#### Ingress class proposal
 
-Promoting the annotation as is as an opaque string the most direct path, but
-precludes any future enhancements to the concept.
+Promoting the annotation as it is currently defined as an opaque string is the
+most direct path but precludes any future enhancements to the concept.
```

Comment:
It is likely that the annotation based IngressClass support is not going away anytime soon even after the v1 release.
To minimize differences in implementations, we should define explictly the interoperability of annotation driven vs this approach, such as this field takes precedence over the annotation.

Reply:
I added a section on interoperability, please take a look.